R/session-info.r

```diff
@@ -153,10 +153,27 @@ pkg_source <- function(desc) {
                   desc$GithubRepo, "@",
                   substr(desc$GithubSHA1, 1, 7), ")")
   } else if (!is.null(desc$RemoteType)) {
-    str <- paste0(desc$RemoteType, " (",
-      desc$RemoteUsername, "/",
-      desc$RemoteRepo, "@",
-      substr(desc$RemoteSha, 1, 7), ")")
+    remote_type <- desc$RemoteType
+
+    user_repo <- character(0)
```

Comment:
This definition (and `sha` and `user_sha` below) would be better off in an else clause.